packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -305,19 +304,16 @@ const AutoBullet: BuildInEditFeature<PluginKeyboardEvent> = {
             true /*canUndoByBackspace*/
         );
     },
+    defaultDisabled: true,
```

Comment:
Besides this, I think you can now clear the content of this feature, keep the feature itself but make it do nothing. So that 
1. Even it is added, it will not be conflict with the new feature
2. Save some bytes.